### PR TITLE
DOC: installation

### PR DESCRIPTION
**Pull request by UranusSeven**
## Related issue number

Fixes #xxxx

## Check code requirements

- [x] tests added / passed (if needed)
- [x] Ensure all linting tests pass

### CODE REVIEW

**Review by aresnow1 — DISMISSED** [review]
LGTM

File: doc/source/getting_started/installation.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+sqlalchemy                                                       1.2.0
+defusedxml                                                       0.5.0
+tqdm                                                             4.1.0
+uvloop (for systems other than win32)                            0.14.0
+================================================================ ==========================
+
+Recommended dependencies
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+* `numexpr <https://github.com/pydata/numexpr>`__: for accelerating certain numerical operations.
```
Comment (qianduoduo0904) [inline doc/source/getting_started/installation.rst:42]: We can tell user simply using `pip install 'xorbits[extra]'` to install al the recommendations.

Comment (UranusSeven) [reply doc/source/getting_started/installation.rst:51]: Good point!

**Review by qianduoduo0904 — APPROVED** [review]
LGTM

**Review by aresnow1 — APPROVED** [review]
LGTM

**Pull request merged**